### PR TITLE
Add angle

### LINKED ISSUE #7809: angle

### PR DESCRIPTION
**Pull request by xezbeth**
Closes #7809

### CODE REVIEW

**Comment by xezbeth** [comment]
angle #7809

**Comment by xezbeth** [comment]
Closes #7809

**Comment by fspyridakos** [comment]
Hey @xezbeth thanks for your contribution!
Could you please remove the files from the .idea folder from your PR? Those should not be getting merged. Reffer to this thread for an explanation as to how this can be done: https://stackoverflow.com/questions/38743912/remove-a-file-from-a-git-pull-request

**Comment by xezbeth** [comment]
> Hey @xezbeth thanks for your contribution! Could you please remove the files from the .idea folder from your PR? Those should not be getting merged. Reffer to this thread for an explanation as to how this can be done: https://stackoverflow.com/questions/38743912/remove-a-file-from-a-git-pull-request

I reverted the .idea folder. Don't know how these changes where made in the first place though.

**Review by xezbeth — COMMENTED** [review]
update tensorflow

File: ivy/functional/backends/tensorflow/experimental/elementwise.py [hunk 1]
```diff
@@ -307,15 +306,26 @@ def diff(
...
 
+@with_unsupported_dtypes({"2.9.1 and below": ("uint8", "uint16", "uint32", "uint64", "bfloat16", "int32")}, backend_version)
+def angle(
+        input: Union[tf.Tensor, tf.Variable],
+        /,
+        *,
+        deg: Optional[bool] = None,
+        out: Optional[Union[tf.Tensor, tf.Variable]] = None,
+) -> Union[tf.Tensor, tf.Variable]:
+    return tf.math.angle(input, name=None)
```
Comment (fspyridakos) [inline ivy/functional/backends/tensorflow/experimental/elementwise.py:326]: I don't see the `deg` parameter being handled here, you should have a check if `deg` is True then multiply the function output by 180/pi.

File: ivy/functional/backends/torch/experimental/elementwise.py [hunk 2]
```diff
@@ -225,6 +225,20 @@ def isneginf(
...
 
+def angle(
+    input: torch.Tensor,
+    /,
+    *,
+    deg: Optional[bool] = None,
+    out: Optional[torch.Tensor] = None,
+) -> torch.Tensor:
+
+    return torch.angle(input, out=out)
```
Comment (fspyridakos) [inline ivy/functional/backends/torch/experimental/elementwise.py:236]: `deg` is also not handled here, handle it the same way as I described for tf.

File: ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_elementwise.py [hunk 3]
```diff
@@ -705,6 +705,54 @@ def test_isneginf(
...
+        num_positional_args=num_positional_args,
+        native_array_flags=native_array,
+        container_flags=container_flags,
+        instance_method=instance_method,
+        ground_truth_backend="tensorflow",
+        fw=backend_fw,
+        fn_name=fn_name,
+        on_device=on_device,
+        z=z[0],
+        deg=False,
```
Comment (fspyridakos) [inline ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_elementwise.py:752]: Please pass `deg=deg` once you fix the comments I made for torch and tf, that should make it work

**Review by fspyridakos — CHANGES_REQUESTED** [review]
Hey, please go over the changes I requested and ask another review when done. Also you've introduced a lot of lint changes, mostly in the form of additional indentation, in `tensorflow/experimental/elementwise.py` you can see all of those in the `Files changed` tab, please revert those changes as well.

**Comment by fspyridakos** [comment]
Did some lint fixing and now it's ready to merge! Thanks again for your contribution

**Comment by xezbeth** [comment]
> Did some lint fixing and now it's ready to merge! Thanks again for your contribution

Thank you very much.

**Pull request merged**